app/views/books/_translations.html.erb

```diff
@@ -1,62 +1,39 @@
-<p>
-This book is available in
-  <a href="/book/en">English</a>.
-</p>
-<p>
-  Full translation available in
-  <table>
-  <tr><td><a href="/book/bg">български език</a>,</td></tr>
-  <tr><td><a href="/book/es">Español</a>,</td></tr>
-  <tr><td><a href="/book/fr">Français</a>,</td></tr>
-  <tr><td><a href="/book/ja">日本語</a>,</td></tr>
-  <tr><td><a href="/book/ko">한국어</a>,</td></tr>
-  <tr><td><a href="/book/nl">Nederlands</a>,</td></tr>
-  <tr><td><a href="/book/ru">Русский</a>,</td></tr>
-  <tr><td><a href="/book/uk">Українська</a></td></tr>
-  <tr><td><a href="/book/zh">简体中文</a>,</td></tr>
-  </table>
-</p>
-<p>
-  Partial translations available in
-  <table>
-    <tr><td><a href="/book/cs">Čeština</a>,</td></tr>
-  <tr><td><a href="/book/id">Indonesian</a>,</td></tr>
-<!--  <a href="/book/fi">Suomi</a>, -->
-  <tr><td><a href="/book/pl">Polski</a>,</td></tr>
-  <tr><td><a href="/book/sr">Српски</a>,</td></tr>
-  <tr><td><a href="/book/tl">Tagalog</a>,</td></tr>
-  <tr><td><a href="/book/zh-tw">繁體中文</a>,</td></tr>
-  </table>
-</p>
-<p>
-  Translations started for
-  <table>
-<!--  <tr><td><a href="/book/ar">Arabic</a>,</td></tr>
-   <a href="/book/az">Azərbaycan dili</a>, -->
-  <tr><td><a href="/book/be">Беларуская</a>,</td></tr>
-<!--  <a href="/book/ca">Català</a>, -->
-   <tr><td><a href="/book/de">Deutsch</a>,</td></tr>
-<!-- <a href="/book/eo">Esperanto</a>,
-     <a href="/book/es-ni">Español (Nicaragua)</a>, -->
-  <tr><td><a href="/book/fa" dir="rtl">فارسی</a>,</td></tr>
-  <tr><td><a href="/book/gr">Ελληνικά</a>,</td></tr>
-<!--  <a href="/book/hi">हिन्दी</a>,
-  <a href="/book/hu">Magyar</a>, -->
-  <tr><td><a href="/book/it">Italiano</a>,</td></tr>
-  <tr><td><a href="/book/mk">Македонски</a>,</td></tr>
-<!--  <a href="/book/no-nb">Norwegian Bokmål</a>, -->
-  <tr><td><a href="/book/ms">Bahasa Melayu</a>,</td></tr>
-  <tr><td><a href="/book/pl">Polski</a>,</td></tr>
-  <tr><td><a href="/book/pt-br">Português (Brasil)</a>,</td></tr>
-<!--  <a href="/book/ro">Română</a>,
-  <tr><td><a href="/book/th">ภาษาไทย</a>,</td></tr> -->
-<!--  <a href="/book/vi">Tiếng Việt</a>, -->
-  <tr><td><a href="/book/tr">Türkçe</a>,</td></tr>
-<tr><td><a href="/book/uz/v2">Ўзбекча</a>.</td></tr>
-</table>
-</p>
-<hr class="sidebar"/>
-<p>
-The source of this book is <a href="https://github.com/progit/progit2">hosted on GitHub.</a></br>
-Patches, suggestions and comments are welcome.
-</p>
+<form class="js-book-lang pv2 " method="get" action="#">
```

Comment:
this list needs to be templated instead of hardcoded right?

Reply:
Not sure I follow. The list is already hardcoded in the partial. I just moved each option into this `select` dropdown.

Reply:
oh, right.  overlooked. I don't see any reason for not using the info from the DB. but we can add an issue to see that afterwards

Reply:
I'm hoping we can :fire: the database at some point in the future, so I'm all for not relying on it more than we already are.

Reply:
Yeah, I think it is OK to just keep it in the template source, as long as there is only one such location.